core/src/main/java/com/amazon/opendistroforelasticsearch/sql/data/model/ExprValue.java

```diff
@@ -31,6 +31,11 @@
    */
   ExprType type();
 
+  /**
+   * Compare the Object value of itself to the ExprValue v.
+   */
+  int compareTo(ExprValue v);
```

Comment:
How does this interface been used?

Reply:
To compare the values of instances of the ExprValue subclasses exactly like basic java objects, for example, the comparison between`ExprIntegerValue` instances like 
```
ExprIntegerValue v1 = new ExprIntegerValue(1);
ExprIntegerValue v2 = new ExprIntegerValue(2);
```
Then the comparison between v1 and v2 has the same behavior of the comparison between integers 1 and 2, aka:
```
v1.compareTo(v2) -> -1
v2.compareTo(v1) -> 1
v1.compareTo(v1) -> 0
```